src/AutoPilotPlugins/PX4/PX4TuningComponentPlaneAttitude.qml

```diff
@@ -0,0 +1,88 @@
+/****************************************************************************
+ *
+ * (c) 2021 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+import QtQuick          2.3
+import QtQuick.Controls 1.2
+import QtQuick.Layouts  1.2
+
+import QGroundControl               1.0
+import QGroundControl.Controls      1.0
+import QGroundControl.FactSystem    1.0
+import QGroundControl.FactControls  1.0
+import QGroundControl.ScreenTools   1.0
+import QGroundControl.Vehicle       1.0
+
+ColumnLayout {
+    width: availableWidth
+    anchors.fill: parent
+    property alias autotuningEnabled: pidTuning.autotuningEnabled
+
+    PIDTuning {
+        width: availableWidth
+        id:    pidTuning
+
+        property var roll: QtObject {
+            property string name: qsTr("Roll")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.roll.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.roll.value }
+            ]
+            property var params: ListModel {
+                ListElement {
+                    title:          qsTr("Time constant (FW_R_TC)")
+                    description:    qsTr("The latency between a roll step input and the achieved setpoint (inverse to a P gain)")
+                    param:          "FW_R_TC"
+                    min:            0.4
+                    max:            1.0
+                    step:           0.1
+                }
+            }
+        }
+        property var pitch: QtObject {
+            property string name: qsTr("Pitch")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.pitch.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.pitch.value }
+            ]
+            property var params: ListModel {
+                ListElement {
+                    title:          qsTr("Time Constant (FW_P_TC)")
+                    description:    qsTr("The latency between a pitch step input and the achieved setpoint (inverse to a P gain)")
+                    param:          "FW_P_TC"
+                    min:            0.2
+                    max:            1.0
+                    step:           0.1
+                }
+            }
+        }
+        property var yaw: QtObject {
+            property string name: qsTr("Yaw")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.heading.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.yaw.value }
+            ]
+            // property var params: ListModel {
```

Comment:
Did you want to add yaw as well or completely remove it?

Reply:
I wanted to add yaw as well but since yaw control is not tuned as a separate parameter, I just wanted the response to be visualized

Reply:
I'm wondering if adding the yaw "response" here may do more harm than good, i.e. folks thinking the signals need to line up, when in reality yaw is a bit more free for fixed-wing (depends on wind condition).

If we can add in slip control at some point - then would need to add yaw response in for sure. .. actually maybe this is something we could do now with yaw rate being explicitly command-able with your newer revisions for offboard velocity control @Jaeyoung-Lim :) 

wdyt? leave it out for now? or trust it wont confuse folks?

Reply:
@tstastny I removed it from now - for me it was just that I didn't want to hide any information that was available. But I agree this might be confusing for the lay person. We should anyway revise the coordinated control logic and then we can probably add it back